extensions/dicom-microscopy/src/getCommandsModule.ts

```diff
@@ -124,15 +124,15 @@ export default function getCommandsModule({
     },
 
     incrementActiveViewport: () => {
-      const { activeViewportIndex, viewports } = viewportGridService.getState();
-      const nextViewportIndex = (activeViewportIndex + 1) % viewports.length;
-      viewportGridService.setActiveViewportIndex(nextViewportIndex);
+      const { activeViewportId, viewports } = viewportGridService.getState();
+      const nextViewportIndex = (activeViewportId + 1) % viewports.size;
```

Comment:
This seems wrong - you should need to find the index and then add to it.  Suggest implementing this once, with a delta value and then set the delta appropriately.  Alternatively, add a viewportGridService.getActiveIndex() function that does this, and perhaps the setActiveIndex,  just as functions that apply correctly.

Reply:
I really don't want to give people the ability to rely on index again, the purpose of this PR is to really remove viewportIndex once for all